index.html

```diff
@@ -66,7 +66,7 @@ <h1>Tonic</h1>
           <li>css</li>
           <li>javaScript</li>
         </ul>
-        <button>See Project</button>
+        <button onclick="showPopup(0)">See Project</button>
```

Comment:
- [ ] At the moment, when you click on the `See project` button nothing happens. Please ensure that all works fine before requesting for another review.